.travis.yml

```diff
@@ -9,12 +9,14 @@ julia:
   - 1.0
   - nightly
 
+services: docker
```

Comment:
It feels like it would be better to have all the PDF-related stuff living in DocumenteLaTeX, including testing the Docker image. But with the writer living here currently, we can keep the testing here until it's moved.